client/my-sites/stats/site.jsx

```diff
@@ -485,16 +501,33 @@ class StatsSite extends Component {
 								className={ classNames(
 									{
 										'stats__author-views': ! supportsUTMStats,
-										'stats__flexible-grid-item--one-third--two-spaces': ! supportsUTMStats,
-										'stats__flexible-grid-item--half--large': ! supportsUTMStats,
 									},
+									'stats__flexible-grid-item--half',
+									'stats__flexible-grid-item--full--medium'
+								) }
+								showSummaryLink
+							/>
+						) }
+
+						{ /* Either stacks with "Authors" or takes full width, depending on UTM and Authors visibility */ }
+						{ supportsEmailStats && (
```

Comment:
Emails and Authors are now reversed so that the Emails module can take full width in case Authors is hidden to help align the following modules.